Faraday_Proxy_Tools/FaradayIO/deviceconfig.py

```diff
@@ -163,13 +175,70 @@ def update_gps(self, gps_boot_bitmask, latitude_str, latitude_dir_str, longitude
 
         :return: Nothing
         """
+        # Setup check variables
+        lat_check = True
+        lat_dir_check = True
+        lon_check = True
+        lon_dir_check = True
+        alt_check = True
+        alt_units_check = True
+
+        # Check Latitude formatting
+
         lat_check = len(latitude_str) <= self.MAX_GPS_LATITUDE_LEN
         lat_dir_check = len(latitude_dir_str) <= self.MAX_GPS_LATITUDE_DIR_LEN
```

Comment:
Not totally ridiculous. The GPS Bug #63 was due to incorrect user input validation and therefore scrutinizing user input for GPS variables seems legitimate. Totally OK to push to later with an Issue Ticket but yeah...